src/entities/Task/model/slice/tasksSlice.ts

```diff
@@ -0,0 +1,87 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import {
+    StatusEnum, SubTask, Task, TasksSchema,
+} from '../types/task';
+
+const initialState: TasksSchema = {
+    data: [],
+};
+
+const updateLocalStorage = (tasks: Task[]) => {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const getTaskById = (tasks: (Task | SubTask)[], id: string) => tasks.find((task) => task.id === id);
```

Comment:
use generics here, so you don't have to use `as Task` or `as SubTask` after each call below

Reply:
Done